app/src/androidTest/java/com/github/se/signify/ui/screens/MainActivityTest.kt

```diff
@@ -33,15 +33,14 @@ class MainActivityTest {
   @Before
   fun setUp() {
     val userRepository = mock(UserRepository::class.java)
-    val userViewModel = UserViewModel(userRepository)
     val context = mock(Context::class.java)
 
     composeTestRule.setContent {
-      FriendsListScreen(navigationActions, userViewModel)
-      SettingsScreen(navigationActions, userViewModel)
-      ProfileScreen(navigationActions, userViewModel)
+      FriendsListScreen(navigationActions, userRepository)
+      SettingsScreen(navigationActions, userRepository)
+      ProfileScreen(navigationActions, userRepository)
       // Set the content with the mocked context
```

Comment:
Is it necessary to add the `FriendsListScreen()`, `SettingsScreen()` and `ProfileScreen()` here as they are already called with `SignifyAppPreview()`.

Reply:
I haven't touched this test. The function calls have only been updated with their new parameters.